domain/src/main/java/onlydust/com/marketplace/api/domain/service/ProjectService.java

```diff
@@ -65,4 +67,23 @@ public UUID createProject(CreateProjectCommand command) {
     public URL saveLogoImage(InputStream imageInputStream) {
         return this.imageStoragePort.storeImage(imageInputStream);
     }
+
+    @Override
+    public Page<ProjectContributorsLinkView> getContributors(UUID projectId,
+                                                             ProjectContributorsLinkView.SortBy sortBy,
+                                                             Integer pageIndex, Integer pageSize) {
+        return projectStoragePort.findContributors(projectId, sortBy, pageIndex, pageSize);
+    }
+
+    @Override
+    public Page<ProjectContributorsLinkView> getContributorsForProjectLeadId(UUID projectId,
```

Comment:
```suggestion
    public Page<ProjectContributorsLinkView> getContributorsForUser(UUID projectId,
```
I would rename it this way because you don't know if the user is a lead or not when you call this method. Plus, this method actually works when the user is NOT a lead (by defaulting on `findContributors`)